tests/errors.test.ts

```diff
@@ -1,13 +1,13 @@
 import fs from 'fs';
 import path from 'path';
 import { zkVerifySession } from '../src';
-import { ZkVerifyEvents } from "../src/enums";
+import { ZkVerifyEvents } from "../src";
 
 jest.setTimeout(180000);
 
 describe('verify with bad data - Groth16', () => {
     it('should fail when sending groth16 data that cannot be formatted and emit an error event', async () => {
-        const dataPath = path.join(__dirname, 'data', 'groth16_error.json');
+        const dataPath = path.join(__dirname, 'common/data', 'groth16_error.json');
```

Comment:
Also update in line 53

Reply:
Doh! Thanks yes this was originally just for groth16 but needs changing to be all proof types

Reply:
Fixed.